docs/jwst/associations/level2_asn_technical.rst

```diff
@@ -29,75 +27,79 @@ Example Association
+        "asn_id": "o037",
+        "asn_pool": "jw00623_20210610t121508_pool",
+        "target": "9",
         "products": [
             {
-                "name": "test_lrs1",
+                "name": "jw00623037001_02101_00001_mirimage",
                 "members": [
                     {
-                        "expname": "test_lrs1_rate.fits",
-                        "exptype": "science"
-                    }
-                ]
-            },
-            {
-                "name": "test_lrs2bkg",
-                "members": [
-                    {
-                        "expname": "test_lrs2bkg_rate.fits",
-                        "exptype": "science"
-                    }
-                ]
-            },
-            {
-                "name": "test_lrs2",
-                "members": [
-                    {
-                        "expname": "test_lrs2_rate.fits",
-                        "exptype": "science"
-                    },
-                    {
-                        "expname": "test_lrs2bkg_rate.fits",
-                        "exptype": "background"
+                        "expname": "jw00623037001_02101_00001_mirimage_rate.fits",
+                        "exptype": "science",
+                        "exposerr": "null"
                     }
                 ]
             }
         ]
     }
 
+.. _asn-level2-meta-keywords:
+
 Association Meta Keywords
 -------------------------
 
-The following are the top-level, or meta, keywords of an association.
+The following are the informational, meta keywords of an association.
 
-program *optional*
-  Program number for which this association was created.
-  
-asn_type *optional*
-  The type of association represented. See :ref:`asn-jwst-association-types`
-
-asn_id *optional*
+asn_id *required*
   The association id. The id is what appears in the :ref:`asn-jwst-naming`
   
-asn_pool *optional*
+asn_pool *required*
   Association pool from which this association was created.
 
 asn_rule *optional*
   Name of the association rule which created this association.
-  
-version_id *optional*
-  Version identifier. DMS uses a time stamp with the format
-  ``yyyymmddthhmmss``
-  Can be None or NULL
+
+asn_type *optional*
+  The type of association represented. See :ref:`asn-jwst-association-types`
+
+code_version *optional*
+  The version of the generator which created this association. Typically this is the version
```

Comment:
which -> that

 "that" is used when the following phrase is needed to uniquely identify the preceding object, e.g. the house that is on the corner is blue. "which" is used when the following phrase is parenthetical (e.g. additional, but unnecessary information), e.g. the house on the corner, which is blue, got hit by lighting last night.

Thus endeth the lesson ...

Reply:
✅